resources/deu.ini

```diff
@@ -186,31 +186,31 @@ give.received={%0} gab dir das Grundstück: {%1}
 ; The /p warp Subcommand
 warp.name=warp
 warp.desc=Teleportiere dich zu einem bestimmten Grundstück.
-warp.usage=/p warp <id: int> [world: string]
+warp.usage=/p warp <id: int> [welt: string]
 warp.wrongid="Die Grundstück ID sollte in diesem Format sein: X;Z"
 warp.notinplotworld=Du bist nicht in einer Grundstückswelt!
 warp.unclaimed=Du kannst dich nicht zu einem Grundstück teleportieren, das niemanden gehört.
 warp.success=Du wurdest zu dem Grundstück {%0} teleportiert!
 
 ; The /p middle Subcommand
 middle.name=middle
-middle.alias=mid
+middle.alias=middle
```

Comment:
Needs reverted. Mid is the alias

Reply:
Oh lol, I thougt I edit something else 🤦‍♂